### PR TITLE
Implement dotenv file

### PR DESCRIPTION
**Pull request by juliekohl**

### CODE REVIEW

File: src/components/CoworkerRegister.vue [hunk 1]
```diff
@@ -119,15 +119,15 @@ export default {
   },
   methods: {
     handleCreate(_this: any) {
-      axios.post("http://localhost:3000/coworkers", _this.user);
+      axios.post(`${process.env.VUE_APP_BACKEND_URL}/coworkers`, _this.user);
       location.reload();
     },
     handleEdit(_this: any) {
-      axios.post(`http://localhost:3000/coworkers/${_this.coworker.id}`, _this.user);
+      axios.post(`${process.env.VUE_APP_BACKEND_URL}/${_this.coworker.id}`, _this.user);
```
Comment (cesarkohl) [inline src/components/CoworkerRegister.vue:126]: Add `/coworkers`

File: src/components/CoworkerRegister.vue [hunk 2]
```diff
@@ -119,15 +119,15 @@ export default {
...
       location.reload();
     },
     handleDelete(_this: any) {
-      axios.delete(`http://localhost:3000/coworkers/${_this.coworker.id}`);
+      axios.delete(`${process.env.VUE_APP_BACKEND_URL}/${_this.coworker.id}`);
```
Comment (cesarkohl) [inline src/components/CoworkerRegister.vue:130]: Add `/coworkers`

File: src/pages/Coworkers/Coworkers.vue [hunk 3]
```diff
@@ -34,8 +34,14 @@ export default defineComponent( {
     const all = ref({});
     const coworker = ref({id: null, name: ''});
     const coworkersId = coworker.value.id;
+    const coworking = ref({id: 5});
+    console.log('coworking', coworking);
```
Comment (cesarkohl) [inline src/pages/Coworkers/Coworkers.vue:38]: Remove log

File: src/pages/Coworkers/Coworkers.vue [hunk 4]
```diff
@@ -34,8 +34,14 @@ export default defineComponent( {
...
 
-    fetch('http://localhost:3000/coworkers?coworker_id=5')
+    fetch(`${process.env.VUE_APP_BACKEND_URL}/${coworking.value}`)
```
Comment (cesarkohl) [inline src/pages/Coworkers/Coworkers.vue:40]: Add `/coworkers`

**Review by cesarkohl — APPROVED** [review]
Pre-approved!

File: src/components/CoworkerRegister.vue (hunk 1 above)
Comment (juliekohl) [reply src/components/CoworkerRegister.vue:126]: Done.

File: src/components/CoworkerRegister.vue (hunk 2 above)
Comment (juliekohl) [reply src/components/CoworkerRegister.vue:130]: Done.

File: src/pages/Coworkers/Coworkers.vue (hunk 3 above)
Comment (juliekohl) [reply src/pages/Coworkers/Coworkers.vue:38]: Done.

File: src/pages/Coworkers/Coworkers.vue (hunk 4 above)
Comment (juliekohl) [reply src/pages/Coworkers/Coworkers.vue:40]: Dane.

**Pull request merged**